roles/hitchwiki/tasks/system.yml

```diff
@@ -0,0 +1,64 @@
+---
+# Common tasks
+
+- name: 'install python2' # Ansible needs python or it fails with "/usr/bin/python: not found"
+  become: yes
+  raw: apt-get install -yq python-simplejson
+  # See https://stackoverflow.com/questions/32429259/ansible-fails-with-bin-sh-1-usr-bin-python-not-found
+
+- debug:
+    msg: "root: {{ dir.root }}"
+
+- name: Has configs/settings-example.yml been copied to configs/settings.yml?
+  include_vars: ./configs/settings.yml
+  when: lookup('file', 'configs/settings.yml')
+
+- name: Set language
+  lineinfile: 
+    path: .profile
+    line: |
+      export LC_ALL="en_US.UTF-8"
+      export LANGUAGE="en_US.UTF-8"
+
+- name: Use apt-cacher
```

Comment:
What's this used for?

Reply:
apt proxy that caches apt files, useful for slow internet when you repeatedly destroy your box